airbyte-workers/src/main/java/io/airbyte/workers/DefaultSyncWorker.java

```diff
@@ -145,7 +145,29 @@ public StandardSyncOutput run(StandardSyncInput syncInput, Path jobRoot) throws
 
   @Override
   public void cancel() {
+    LOGGER.info("Cancelling sync worker...");
     cancelled.set(true);
+
+    LOGGER.info("Cancelling source...");
+    try {
+      source.cancel();
```

Comment:
curious: what was cancelling the source/destination prior to this change?

Reply:
nothing